### ISSUE TITLE
Student name/email in git history

### ISSUE THREAD

**Issue by pstoeckle** [ls1intum/Artemis#2957]
### Is your feature request related to a problem?

![13e15](https://user-images.githubusercontent.com/5854768/109852918-9cd1ed80-7c55-11eb-8bda-643dc4424539.png)

When I click on the `Download Repo` button and unzip the dowloaded folder, I can execute git commands on the student's repository, e.g., 

```bash
$ git logs 
commit 0de45bf82d19a4e6fc64afb7bebe406501b2189e (HEAD -> diff)
Author: M... M... <...@tum.de>
Date:   Wed Mar 3 18:41:55 2021 +0100

    All student changes in one commit
```

What I really like about Artemis is that the assignments are anonymized, but through downloading the assignment, I can break this anonymity very easily. Now I am wondering: Is there a reason for this? Or is there a switch somewhere to activate an anonymization also for the git history? 

### Describe the solution you'd like

When clicking on `export`, Artemis is either
* recreating the history with some placeholders for name and email, e.g., `Max Mustermann` and `max.mustermann@tum.de` or 
* creating one single commit with the placeholders

**Comment by krusche** [comment]
You are right, this is one of the few places where the anonymity is not yet fully realized. We will consider this in the future.
What we cannot really fix is the "Go to repository" button. Tutors can navigate to Bitbucket and review the student repository over there. This is sometimes really helpful to help the students in case of issues, so I am not sure if we should prevent this in the future.
